inference-engine/ie_bridges/python/wheel/setup.py

```diff
@@ -87,6 +87,12 @@
         'install_dir': TBB_LIBS_DIR,
         'rpath': LIBS_RPATH,
     },
+    'cpu_extensions': {
+        'name': 'template_extension',
```

Comment:
@dkurt Why do you add `template_extension` to python package? We shouldn't do it, `template_extension` is used only in OpenVINO documentation.

Reply:
Our goal is to enable the demo https://github.com/openvinotoolkit/openvino_notebooks/pull/46. At this moment there is no other alternative to make it work but MO+nGraph+plugins enablement